squad/frontend/templates/squad/project.html

```diff
@@ -8,13 +8,12 @@
 
 {% include "squad/project-nav.html" %}
 
-{% if last_build %}
```

Comment:
this conditional should stay. The group might not have any builds.

Reply:
Fixed. Will have it ready for the next pull request